### PR TITLE
Ensure that token cannot be tested more than 3 times

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [ ] I've added a mention in CHANGELOG.md
   ^--- Unit test only, not relevant

### CODE REVIEW

File: tests/routes/jwt.test.mjs [hunk 1]
```diff
@@ -50,6 +50,59 @@ describe('Test the JWT route', () => {
     });
   });
 
+  it('It should gain access on generate via uniq token', async () => {
+    const PASSWORD = 'foobarbaz';
```
Comment (Scttpr) [inline tests/routes/jwt.test.mjs]: I would use beforeEach and afterEach to avoid code repetition here

File: tests/routes/jwt.test.mjs [hunk 2]
```diff
@@ -50,6 +50,59 @@ describe('Test the JWT route', () => {
...
+          email: user.email,
+          token: resetToken,
+        });
+      expect(statusCode).to.equal(200);
+      expect(
+        jwt.decode(token, { secret: config.get('token:secret') }),
+      )
+        .to.deep.include({ email: user.email, id: user.id });
+    } finally {
+      await user.deleteOne();
```
Comment (Scttpr) [inline tests/routes/jwt.test.mjs]: I would use beforeEach and afterEach to avoid code repetition here

File: tests/routes/jwt.test.mjs [hunk 3]
```diff
@@ -50,6 +50,59 @@ describe('Test the JWT route', () => {
...
+
+  it('It should block access on generate via token after 3 failures', async () => {
+    const MAX_RETRY = 3;
+    const PASSWORD = 'foobarbaz';
+    const user = await createDummyUser({ password: PASSWORD });
+    const { token } = await user.generateResetToken({ email: user.email });
+    await user.save();
+    try {
+      // eslint-disable-next-line no-restricted-syntax
+      for (const i of [...Array(MAX_RETRY).keys()]) {
```
Comment (Scttpr) [inline tests/routes/jwt.test.mjs]: Here `let`is not compulsory because `i` is not modified during iteration ?

Comment (Scttpr) [inline tests/routes/jwt.test.mjs]: Is there a specific reason to use a `for...of` here ?

File: tests/routes/jwt.test.mjs (hunk 2 above)
Comment (Nainterceptor) [reply tests/routes/jwt.test.mjs]: We don't have only 2 tests in the file

File: tests/routes/jwt.test.mjs (hunk 1 above)
Comment (Nainterceptor) [reply tests/routes/jwt.test.mjs]: We don't have only 2 tests in the file

File: tests/routes/jwt.test.mjs (hunk 3 above)
Comment (Nainterceptor) [reply tests/routes/jwt.test.mjs]: scope of `i` start in the loop

Comment (Nainterceptor) [reply tests/routes/jwt.test.mjs]: To be able to await

**Pull request merged**